salt/states/cmd.py

```diff
@@ -171,7 +171,7 @@
 
 .. note::
 
-    Use ``cmd.run`` together with :mod:`onchanges </ref/states/requisites#onchanges>`
+    Use :mod:`cmd.run <salt.states.cmd.run>` together with :mod:`onchanges </ref/states/requisites#onchanges>`
```

Comment:
I know that you didn't write the original code here for the ``onchanges`` link, but this is an invalid link, and ``:mod:`` isn't used for user-generated anchors anyway, so I don't think this link _ever_ worked.

To get this link fixed, please change this to:
```
:ref:`onchanges <requisites-onchanges>`
```

That will link it to the user-defined anchor for the ``onchanges`` requisite in **doc/ref/states/requisites.rst**.